newa/cli.py

```diff
@@ -114,7 +114,7 @@ def cmd_event(ctx: CLIContext, errata_ids: tuple[str, ...]) -> None:
             releases = ['RHEL-8.10.0', 'RHEL-9.4.0']
 
             for release in releases:
-                erratum_job = ErratumJob(event=erratum.event, erratum=Erratum(release=release))
+                erratum_job = ErratumJob(event=erratum.event, erratum=Erratum(release=release, builds=[]))
```

Comment:
If you define `builds` with a default value, `builds: list[str] = dataclasses.field(default_factory=list)`, you may omit `builds=[]` here, an empty list will be constructed automagically.

Reply:
Right, I think I tried that at some point but did it wrong way. I'll fix that.